templates/terraform/examples/base_configs/documentation.tf.erb

```diff
@@ -1,3 +1 @@
-```hcl
```

Comment:
How does this affect normal TF?

Reply:
It doesn't, because I wrapped the only usage (at https://github.com/GoogleCloudPlatform/magic-modules/pull/2279/files#diff-5f9e88fdf73ae3f58c229d7188538667R94)

This PR removes an extraneous newline that I noticed, because of https://github.com/GoogleCloudPlatform/magic-modules/pull/2279/files#diff-5f9e88fdf73ae3f58c229d7188538667L93